Contributing.md

```diff
@@ -39,6 +47,7 @@ Current Issue Triagers:
 Our most regular and experienced Issue Triagers sometimes move on to doing code reviews for pull requests, and have input into which pull requests should be merged. 
 
 Current Reviewers:
+- [@hwillson](https://github.com/hwillson)
 - [@lorensr](https://github.com/lorensr)
 - [@abernix](https://github.com/abernix)
```

Comment:
Just to be super clear @hwillson I was not questioning your addition, just whether Jesse meant to leave himself here when he added himself to another section.